scripts/fetch/getNMdata.R

```diff
@@ -0,0 +1,12 @@
+#' Get NM daily data
+#'
+#'@param viz the vizlab object 
+fetcher.getNMdata <- function(viz = getContentInfo(viz.id = "getNMdata")){
```

Comment:
I'm setting the default of `viz` to ` getContentInfo(viz.id = "getNMdata")`...which is an AWESOME addition to the package (thanks @aappling-usgs )